File: Weekly/1NovWeek/4_thursday/code/Countdown.java
import java.util.*;

public class Countdown {

  public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    System.out.println("Enter a number: ");
    int countdown = input.nextInt();
    while (countdown >0){
      System.out.println("Countdown");
      countdown--;
    }
    System.out.println("Lift off!");
  }
}

File: Weekly/28-3-22-week/2_tuesday/code/countdown.py
number = int(input("Enter your number here: "))
output = ""
while number > 0:
    output += str(number) + "..."
    number -= 2
print(output + "Blastoff!")

number = int(input("Enter your number here: "))
def countdown(number):
    if number <= 0:
        return "Blastoff!"
    else:
        return str(number) + "..." + countdown(number - 2)
print(countdown(number))
